app/controllers/location_controller.rb

```diff
@@ -0,0 +1,9 @@
+class LocationController < ApplicationController
+  def index
+    if params[:search].present?
+      @locations = Location.near(params[:search], 50, :order => :distance)
```

Comment:
Please do not use old syntax 

```ruby
:order => :distance
```

```ruby
order: distance
```

But also, what is `:distance` ? Where is it defined?